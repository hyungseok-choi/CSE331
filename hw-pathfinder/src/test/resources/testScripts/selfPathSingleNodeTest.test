CreateGraph simpleGraph

AddNode simpleGraph A

FindPath simpleGraph A A
