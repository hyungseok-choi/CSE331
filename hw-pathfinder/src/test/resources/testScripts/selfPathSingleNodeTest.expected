created graph simpleGraph

added node A to simpleGraph

path from A to A:
total cost: 0.000
